sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExpression.scala

```diff
@@ -166,6 +166,9 @@ class GpuWindowExpressionMeta(
       childExprs.head.convertToGpu(),
       childExprs(1).convertToGpu().asInstanceOf[GpuWindowSpecDefinition]
     )
+
+  override def isSupportedType(t: DataType): Boolean =
```

Comment:
Do we need to add some extra checks into `tagExprForGpu` so that we only allow it in the cases we care about/tested?  Specifically I mean for PythonUDFs. We are probably okay for now because we did not allow arrays for the regular `GpuWindowExec`, but it might be nice to put in a comment about this.

Reply:
Yes we need. Updated to allow array type only for Python UDF.

Reply:
Yes, we do, i am working on this